tests/wsl/update_windows.pm

```diff
@@ -7,24 +7,40 @@
 # Maintainer: qa-c <qa-c@suse.de>
 
 use Mojo::Base qw(windowsbasetest);
-use testapi qw(assert_and_click assert_screen check_screen match_has_tag);
+use testapi;
 
 sub run {
     my $self = shift;
-    $self->windows_run('control update');
 
-    assert_screen 'windows-update';
-    while (defined(check_screen('windows-updates-available', 60))) {
-        bmwqemu::diag("Updating windows base image file...");
-        sleep 30;
+    my $vbs_url = data_url("wsl/UpdateInstall.vbs");
+    $self->open_powershell_as_admin;
+    # Create a registry value in win11 to prevent random reboots
+    if (get_var("WIN_VERSION") == '11') {
+        $self->run_in_powershell(cmd => "reg add HKLM\\SOFTWARE\\Policies\\Microsoft\\Windows /v WindowsUpdate");
+        $self->run_in_powershell(cmd => "reg add HKLM\\SOFTWARE\\Policies\\Microsoft\\Windows\\WindowsUpdate /v AU");
+        $self->run_in_powershell(cmd => "reg add HKLM\\SOFTWARE\\Policies\\Microsoft\\Windows\\WindowsUpdate\\AU /v NoAutoRebootWithLoggedOnUsers /t REG_DWORD /d 1");
```

Comment:
If I could suggest, for a next time, to assign name the repeated constant string like `"reg add HKLM\\SOFTWARE\\Policies\\Microsoft\\Windows\\"` to a parameter and use it in place: IMHO it could help reading the command and be sure that no change done when used more times.